README.md

```diff
@@ -33,26 +32,50 @@ import { enhance, handle } from '@grrr/hansel';
 
 enhance(document.documentElement, {
     enhancer1(elm) {
+        // Enhance element 1
     },
     enhancer2(elm) {
+        // Enhance element 2
     },
     enhancerN(elm) {
+        // Enhance element n
     }
 });
 
 handle(document.documentElement, {
     handler1(elm, event) {
+        // Handle click on element 1
     },
     handler2(elm, event) {
+        // Handle click on element 2
     },
     handlerN(elm) {
+        // Handle click on element n
```

Comment:
It's not really element 1, element 2 etc. though. 
We shouldn't make the impression that there's an enhancer/handler per element. Many elements share the same enhancer. 

I would omit these comments honestly.